lazyflow/operators/ioOperators/opBigTiffReader.py

```diff
@@ -17,55 +16,62 @@ class OpBigTiffReader(Operator):
 
     TIFF_EXTS = [".tif", ".tiff"]
 
-    class NotBigTiffError(Exception):
+    class BigTiffError(Exception):
+        pass
+
+    class NotBigTiffError(BigTiffError):
         """Raised if the file you're attempting to open isn't a BigTiff file."""
 
+    class BigTiffMultiPageError(BigTiffError):
+        """Attemting to load a big tiff with multiple pages, not supported"""
+
+    class BigTiffMultiSeriesError(BigTiffError):
+        """Attemting to load a big tiff with multiple series, not supported"""
+
+    class BigTiffAxistagsError(BigTiffError):
+        """Attemting to load a big tiff with unsupported axistags"""
+
     def __init__(self, *args, **kwargs):
         super(OpBigTiffReader, self).__init__(*args, **kwargs)
         self._bigtiff = None
 
     def cleanUp(self):
-        if self._bigtiff is not None:
-            self._bigtiff.close()
+        # currently the only way to close the memmap
+        del self._bigtiff
         super(OpBigTiffReader, self).cleanUp()
 
     def setupOutputs(self):
-        filepath = self.Filepath.value
-        if not pytiff.utils.is_bigtiff(filepath):
-            raise OpBigTiffReader.NotBigTiffError(filepath)
+        filepath: str = self.Filepath.value
+
+        with tifffile.TiffFile(filepath) as f:
+            if not f.is_bigtiff:
+                raise OpBigTiffReader.NotBigTiffError(filepath)
 
-        bigtiff = pytiff.Tiff(filepath)
+            if f.pages.is_multipage:
+                raise OpBigTiffReader.BigTiffMultiPageError("Multipage BigTiff not supported yet.")
```

Comment:
and yeah, maybe the hierarchy is overdoing it here a little bit. I reduced it to `NotBigTiffError` and `UnsupportedBigTiffError` instead of moving messages to the fine grained exceptions.